tools/reviewer/aa/commands/WorkspaceCommand.java

```diff
@@ -21,21 +21,20 @@
 import com.google.startupos.common.flags.FlagDesc;
 import com.google.startupos.common.flags.Flags;
 import com.google.startupos.tools.reviewer.aa.Protos.Config;
-
-import javax.inject.Inject;
 import java.io.IOException;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.Arrays;
+import javax.inject.Inject;
 
 /**
  * This command is used to switch between workspaces and create new ones.
  *
- * <p>Usage: To switch to a workspace: aa workspace <workspace name>
+ * <p>Usage: To switch to a workspace: aa workspace [workspace name]
```

Comment:
There's a check against `<>`?

Reply:
@oferb it tries to parse it as HTML tag and fails 🙂

Reply:
:)